tests/epyccel/test_loops.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 import numpy as np
 
-from pyccel.epyccel import epyccel
+from pyccel import epyccel
 from modules        import loops
```

Comment:
Alignment is broken, so we can just remove the blank spaces
```suggestion
from pyccel import epyccel
from modules import loops
```